docs/persistence.rst

```diff
@@ -157,38 +166,40 @@ that class. When the ``InnerDoc`` subclass is wrapped with ``List``, a
         ...
     
     class Post(Document):
-        address: Address         # same as Object(Address)
-        comments: List[Comment]  # same as Nested(Comment)
+        address: Address         # same as address = Object(Address)
+        comments: List[Comment]  # same as comments = Nested(Comment)
 
 Unfortunately it is impossible to have Python type hints that uniquely
 identify every possible Elasticsearch field type. To choose a field type that
-is different thant the ones in the table above, the field instance can be added
+is different than the ones in the table above, the field instance can be added
 explicitly as a right-side assignment in the field declaration. The next
 example creates a field that is typed as ``str``, but is mapped to ``Keyword``
 instead of ``Text``:
 
 .. code:: python
 
     class MyDocument(Document):
-        category: str = Keyword()
+        category: str = Keyword(required=True)
 
 This form can also be used when additional options need to be given to
 initialize the field, such as when using custom analyzer settings:
 
 .. code:: python
 
     class Comment(InnerDoc):
-        content: str = Text(analyzer='snowball')
+        content: str = Text(analyzer='snowball', required=True)
```

Comment:
Do we still need this part given the first example already has an option (`required=True`)?

Reply:
Okay, yes, I see that this makes less sense now. I'll convert the previous example to optional, so that this is the first time additional options are added.